### ISSUE TITLE
More fansite URLs (Anti-Adblock sites)

### ISSUE THREAD

**Issue by THEtomaso** [uBlockOrigin/uAssets#3401]
@okiehsch

Here you go..
Like requested; a replacement for the current rule in **uBlock filters** (found under `! https://github.com/DandelionSprout/adfilt/issues/7#issuecomment-418144059`):

`alexandra-daddario.org,alona-tal.com,amandaseyfried.us,amyadams-source.com,anadearmas.net,annakendrick-fr.com,annalise-basso.com,anyataylorjoy.org,archie-panjabi.com,aseyfried.org,avgeropoulos.com,b-larson.net,bellahadid.org,bellathorne.net,beyonceonline.org,brynn-rumfallo.net,callie-hernandez.net,charlize-theron.fr,chloemoretz.org,clemencepoesyfan.com,daisy-ridley.us,daisyridley.us,dove-cameron.org,elizabethlailbrasil.com,elizabetholsen.us,emilyosment.net,emma-roberts.org,emmaengleonline.com,emmawatson.org,fanningdakota.com,fanningelle.com,fans.bz,fansitehost.com,fergie-web.net,florian-silbereisenfan.net,freefansitehosting.com,hailee-steinfeld.org,heidi-klum.fan.pw,jan-smitfan.nl,jenna-coleman.org,jennaortega.net,jennifer-lawrence.com,jennifer-lawrence.us,jenniferlawrence.org,jenniferlawrence.us,jessica-chastain.org,jessica-lowndes.net,jlospain.net,joey-king.net,julia-garner.com,junotemple.us,kaley-cuoco.net,kardashianworld.net,katie-mcgrath.com,katyperryonline.com,kaya-scodelario.org,kendallkyliefans.com,kerirussellweb.com,kiernan-shipka.org,kylierogers.fan.pw,landry-bender.com,lilireinhart.us,lizolsen.com,mackenzie-foy.com,mackenzie-ziegler.net,mackenziefoy.net,maddie-ziegler.net,margotrobbieonline.com,marguerite-moreau.com,mckennagrace.fan.pw,megan-fox.us,meghan-markle.org,michellekeegan.net,mirandakerrfans.com,natalia-dyer.com,olivia-holt.net,olsenfans.com,raffey-cassidy.com,rhona-mitra.com,ritaoracentral.org,rowan-blanchard.net,rowanblanchard.net,ryanphillippe.net,sabrina-carpenter.org,shipka.net,sigourney-weaver.net,simplyleighanne.net,smg-france.net,sofia-carson.org,sofiavergaraweb.net,stellahudgensdaily.net,strahovski.net,vanessa-kirby.com,zooey-deschanel.com##+js(bab-defuser.js)`

If I come by any more of them, I'll post them in this issue thread.

**Comment by THEtomaso** [comment]
```
caitylotz.net
chriswoodfan.com
dianna-agron.org
foreverrobin.com
hilary-swank.com
r-carlylefans.com
```

**Comment by THEtomaso** [comment]
```
chloe-lukasiak.net
daddariofan.net
galleryaloud.com
jade-pettyjohn.com
jamesfranconetwork.net
josh-duhamel.org
k-dunst.net
```

**Comment by THEtomaso** [comment]
```
bellahadid.us
danstevensbrasil.com
depplovers.com.br
jon-bernthal.com
jude-law.net
lizzieolsen.us
lucyhaledaily.us
wentworth-miller.org
```

**Comment by THEtomaso** [comment]
You forgot to include `bellahadid.us` in the last commit.

**Comment by THEtomaso** [comment]
```
hilarie-burton.org
iansomerhalderfan.com
liamhemsworth.org
michael-french.net
mileyupdateuk.com
viggo-mortensen.com
```

**Comment by THEtomaso** [comment]
```
ajlee.org
beckylynchsource.com
believe-in-rollins.com
billiekay.com
drew-mcintyre.com
emma-stone.us
mandy-rose.com
mia-yim.com
niajax.com
trishstratus.com
```

**Comment by ZaphodBeebblebrox** [comment]
Out of curiosity, how do you find all of these sites?

**Comment by THEtomaso** [comment]
There's no science to it.
If you scroll down a bit at any one of those pages, you'll find that most of them comes with a list of "affiliates" (links to other fansites).
This allows you to open a huge amount of different sites in no time.
Then it's just a matter of clicking through all of the tabs, to see if any of them have triggered a anti-adblock warning.

**Comment by THEtomaso** [comment]
```
alliefansite.com
granmetaliksource.com
```

**Comment by THEtomaso** [comment]
```
barbieblanksource.com
inourharts.com
landycannononline.com
summer-rae.com
trinity-fatu.net
```

**Comment by THEtomaso** [comment]
`hailee-steinfeld.com`

**Comment by smed79** [comment]
> Out of curiosity, how do you find all of these sites?

see https://publicwww.com/websites/YWQubWFpbC5ydQ/

**Comment by THEtomaso** [comment]
```
finn-wittrock.net
julianna-margulies.net
```

**Comment by THEtomaso** [comment]
`withlove.info`

**Comment by THEtomaso** [comment]
```
elijah-wood.org
tom-hiddleston.org
```

**Comment by THEtomaso** [comment]
```
bailee-madison.org
lily-chee.com
```

**Comment by THEtomaso** [comment]
```
aj-styles.net
fan.pw
lincedorado.com
mandyrose.org
rey-mysterio.net
```

Example for the `fan.pw` domain:
`http://tessablanchardcom.fan.pw/`

**Comment by THEtomaso** [comment]
`ajmcleanweb.com`

**Comment by THEtomaso** [comment]
```
phillipa-soo.com
violett-beane.com
```

**Comment by okiehsch** [comment]
@smed79 these are not `Fansite URLs` can you open a new issue, it will take quite a while to work through 213 domains and some of them will probably need additional filters.

**Comment by THEtomaso** [comment]
```
eizagonzalez.com.br
paulalacroixbrasil.com
```

**Comment by smed79** [comment]
@okiehsch Sorry for posting here. I deleted my post.

**Comment by THEtomaso** [comment]
`harrypotterweb.com`

**Comment by THEtomaso** [comment]
```
aubreyk-miller.com
ava-allan.com
```

**Comment by THEtomaso** [comment]
`official-makenzie-moss-fanpage.com`

**Comment by THEtomaso** [comment]
```
taron-egerton.com
zoe-kravitz.com
```

**Comment by THEtomaso** [comment]
@mapx-:
You seem to have slipped up a bit there, when adding `taron-egerton.com`.

**Comment by THEtomaso** [comment]
`isabelleallen.net`

**Comment by THEtomaso** [comment]
`missbilliepiper.com`

**Comment by THEtomaso** [comment]
```
lanaparrillabrasil.com
michelle-trachtenberg.net
sweetchin.com
```

**Comment by THEtomaso** [comment]
`maisiewilliams.org`

**Comment by THEtomaso** [comment]
`juliannemooreweb.net`

**Comment by THEtomaso** [comment]
`ariana-grande.org`

PS:
The URL in my previous post hasn't been added yet either.

**Comment by THEtomaso** [comment]
`marion-cotillard.org`

**Comment by THEtomaso** [comment]
```
bella-twins.com
saoirseronan.us
```

**Comment by THEtomaso** [comment]
```
dlovatosource.net
gia-mantegna.com
```

**Comment by THEtomaso** [comment]
```
elizabeth-olsen.org
ellefanning.us
jakegyllenhaal.org
```

**Comment by THEtomaso** [comment]
```
chelseagreen.org
jamie-chung.com
tessa-blanchard.com
troianbellisario.org
```

**Comment by THEtomaso** [comment]
`odette-annable.net`

`##+js(fuckadblock.js-3.2.0)` doesn't seem to work properly for this one.
Might need `##+js(setInterval-defuser.js, visibility, 1000)`

**Comment by mapx-** [comment]
you mean
`odette-annable.net##+js(bab-defuser.js)`

**Comment by THEtomaso** [comment]
Yes, of course.
I was messing around with the other one earlier, and somehow got them mixed up! :laughing:

**Comment by THEtomaso** [comment]
`eddie-redmayne.net`

**Comment by THEtomaso** [comment]
```
dylan-sprayberry.com
kendall-photos.com
```

**Comment by THEtomaso** [comment]
`maggiegracefrance.net`

**Comment by THEtomaso** [comment]
`zoey-deutch.us`

**Comment by THEtomaso** [comment]
```
jameelajamil.tk
jessicachastain.tk
```

**Comment by THEtomaso** [comment]
`vanessaborne.org`

**Comment by THEtomaso** [comment]
`supreme.fan`
(Examples: `iansomerhalderfan.supreme.fan` / `kristenbell.supreme.fan`)

**Comment by THEtomaso** [comment]
`bellatwinsphotos.com`

**Comment by THEtomaso** [comment]
`samheughan-fr.com`

**Comment by THEtomaso** [comment]
```
backalleypics.com
backalleypics.net
```
Dont know if these sites qualifies as "fansites" per se, but they both use BAB.

**Comment by THEtomaso** [comment]
`kiernanshipka.org`

**Comment by THEtomaso** [comment]
```
brandirhodesnet.xyz
haileybaldwin.org
```

**Comment by THEtomaso** [comment]
> https://github.com/uBlockOrigin/uAssets/commit/a1fdc003fff09dc8e468aeccd9602b738475486b¨

What the hell is `##+js(nobab)`??
It broke the whole rule!

**Comment by mapx-** [comment]
https://github.com/gorhill/uBlock/wiki/Resources-Library#nobabjs-

mmhmm are you using legacy stuff ? it wasn't modified like the main uBo

**Comment by THEtomaso** [comment]
Yes, I'm using uBO Legacy.

**Comment by THEtomaso** [comment]
Guess a bunch of new filters will have to be added to '**uBlock filters - Legacy**' then?

**Comment by mapx-** [comment]
I restored the old `##+js(bab-defuser.js)` but other scriptlets were modified ... so in time will be difficult to maintain the compatibility with the legacy version
You should migrate to the some other browser

**Comment by THEtomaso** [comment]
> You should migrate to some other browser

No, what's needed is "Legacy" maintainers.

**Comment by THEtomaso** [comment]
But what about '**uBlock filters - Legacy**', which I mentioned?:
https://github.com/uBlockOrigin/uAssets/blob/master/filters/legacy.txt
For the time being, perhaps all of the existing `##+js(nobab)` rules could be copied into this filter, and changed into `##+js(bab-defuser.js)` rules?

**Comment by mapx-** [comment]
> No, what's needed is "Legacy" maintainers.

> For the time being, perhaps all of the existing rules, which uses the new defuser syntaxes, could be copied into this filter, and changed into using the old syntaxes?

@gorhill @okiehsch

**Comment by gorhill** [comment]
> No, what's needed is "Legacy" maintainers.

I agree: whoever uses uBO legacy is best placed to volunteer to maintain that special list. Asking current volunteers here to bear the burden of a version of uBO which they do not use is unreasonable. If such volunteers come forward, it will be best to maintain the `legacy.txt` in their own repo to avoid undue noise here.

**Comment by okiehsch** [comment]
> For the time being, perhaps all of the existing rules, which uses the new defuser syntaxes, could be copied into this filter, and changed into using the old syntaxes?

Done, this is a one time commit from my end and mybe I missed some filters.
You can open pull requests for the legacy list to ensure the future compatibility if no one volunteers to maintain such a list.

**Comment by THEtomaso** [comment]
Thanks a lot, okiehsch!

**Comment by THEtomaso** [comment]
`darren-criss.net`

**Comment by THEtomaso** [comment]
```
greysanatomy.com.br
johnny-depp.org
```

**Comment by THEtomaso** [comment]
`fanhost.org`

Examples:
```
https://brittbakersource.fanhost.org/
https://simplyhelen.fanhost.org/
http://teamhudgens.fanhost.org/
```

**Comment by THEtomaso** [comment]
```
alexabliss.net
alexablissphotos.com
angelina-love.org
johnnygarganosource.com
theoreillysource.com
```

--

`fanhost.net`

Examples:
```
http://billiekaynet.fanhost.net/
http://crystalreed.fanhost.net/
http://dylansprayberry.fanhost.net/
http://imaginarylight.afan.site/
http://kardashianfashion.fanhost.net/
http://kellykellydiary.fanhost.net/
http://r5daily.fanhost.net/
http://seanharris.fanhost.net/
```

**Comment by THEtomaso** [comment]
```
emma-engle.com
larsenthompson.net
oona-laurence.com
paris-berelc.org
parisberelc.org
rowan-blanchard.org
stefanie-scott.org
zendaya.org
```

**Comment by THEtomaso** [comment]
```
bellahadid.org
beyoncetribe.it
```

@mapx-:
The previous update messed up the entire rule!
You merged the **bab-defuser** rule with the next rule in the list (a **fuckadblock** rule), by removing the line break which separated them!

**Comment by okiehsch** [comment]
I restored the working filter, `bellahadid.org` is already part of it.

**Comment by THEtomaso** [comment]
Yeah, I noticed.

**Comment by THEtomaso** [comment]
`gavinleatherwood.net`

**Comment by THEtomaso** [comment]
`naomigscott.com`

**Comment by THEtomaso** [comment]
`carmen-electra.net`

**Comment by THEtomaso** [comment]
`elsapataky.tk`

**Comment by THEtomaso** [comment]
`meganfoxweb.com`

**Comment by THEtomaso** [comment]
> https://github.com/uBlockOrigin/uAssets/commit/37d361d48aa7785d9b60de74c3ea3e4bc2d20a47¨

I ment "not BAB" as in "not protected by Block Ad Block"..
`##+js(nobab)` doesn't work for `elizabeth-mitchell.org`, but requires a different fix.

**Comment by THEtomaso** [comment]
> https://github.com/uBlockOrigin/uAssets/commit/7c9c43cca0bbc900b822f6ecfd98e775acc2ad9d

I'm not trying to be difficult or anything, but `$ghide `enables some of unwanted elements on that site.
`##+js(aopw, KillAdBlock)` doesn't!

**Comment by THEtomaso** [comment]
> https://github.com/uBlockOrigin/uAssets/commit/422c27ebc98cd5a59f31bbf4aaf6bf0b591c061e

Well, I guess that takes care of the ads.
But still, social media filters and certain annoyance filters are rendered useless by `$ghide` too.
E.g., social media buttons, and that scroll-to-top floater becomes unblocked!
Why so hesitant towards that `##+js` fix?

**Comment by THEtomaso** [comment]
Thanks. :)
Like I said, I'm not trying to be difficult, but there are often more downsides to using $ghide, then some people might think..
For instance, since all anti-cookie filers are rendered useless by it, a cookie warning is displayed at the bottom of that page.
This might not sound very serious, but in this case, the cookie warning's dismiss button seems to use SVG2.0, which is still a drafted spec at this point:
https://www.w3.org/TR/SVG2/

Result (in Pale Moon, and all other browsers that doesn't support that unfinnished standard):

![filterissue-svg2](https://user-images.githubusercontent.com/9382674/87231009-d34fb500-c3b3-11ea-8a0b-847e58acdb71.jpg)

**Comment by THEtomaso** [comment]
`melissabenoist.com.br`

**Comment by THEtomaso** [comment]
`missbilliepiper.com`
(gets triggered in the gallery section @ `https://missbilliepiper.com/gallery/`)

**Comment by THEtomaso** [comment]
`kardashianworld.net`
(gets triggered in the gallery section @ `https://kardashianworld.net/gallery/`)

PS:
I've actually reported this site before (in the first post), but it has since been removed!
A bunch of other sites seems to have been removed too!!
What's the reason for this?

**Comment by mapx-** [comment]
my fault, I checked a lot of pages but not the galleries.

**Comment by THEtomaso** [comment]
Well, I'm not gonna go through all of those sites again.
Do you have a simple way of restoring the ones that you removed?

**Comment by mapx-** [comment]
I'll take a look (it wasn't a single commit)

**Comment by mapx-** [comment]
it should be ok now

**Comment by THEtomaso** [comment]
`les-stars-nues.biz`
Press "ENTRER" to enter the site, and the BAB popup is triggered after a couple of seconds.

**Comment by THEtomaso** [comment]
@mapx-:

> https://github.com/uBlockOrigin/uAssets/commit/55ea908fbfd0fa92bd6721d192185d60b08aca23

All videos are blocked @ `http://www.elizabeth-mitchell.org/video/`

Problem rule: `elizabeth-mitchell.org##+js(aopr, document.body)`

**Comment by THEtomaso** [comment]
`thetvshows.us`

**Comment by THEtomaso** [comment]
@mapx-:
While at it, perhaps you could add this to **uBlock filters - Unbreak** too, because I think okiehsch missed my last ping here:
https://github.com/easylist/easylist/issues/2436#issuecomment-962457361
There are already some entries in the unbreak filer, under a reference to that issue report.

**Comment by okiehsch** [comment]
https://github.com/uBlockOrigin/uAssets/commit/f071791739a7a70ce0cd615be8a0acef7a3c10ed

**Comment by THEtomaso** [comment]
`hqdiesel.net`
